pyop2/codegen/rep2loopy.py

```diff
@@ -460,7 +670,7 @@ def generate(builder, wrapper_name=None):
     # register kernel
     kernel = builder.kernel
     headers = set(kernel._headers)
-    headers = headers | set(["#include <math.h>"])
+    headers = headers | set(["#include <math.h>"]) | set(["#include <stdint.h>"])
```

Comment:
I think that `stdint.h` is now magically included by loopy, but maybe not. If not, probably write this as:

```python
headers |= {"#include <math.h>", "#include <stdint.h>"}
```